README.md

```diff
@@ -129,6 +103,32 @@ running before and after benchmarking to inform you of benchmarking progress),
 which can be specified in the `PRE_EXECUTION_CMDS` and `POST_EXECUTION_CMDS`
 settings in your Krun config file.
 
+For example on a systemd Linux you may turn daemons off before execution with:
+
+```
+PRE_EXECUTION_CMDS = [
+    "sudo systemctl stop cron",
+    "sudo systemctl stop atd",
+    ...
+]
+```
+
+and turn them back on with:
+
+```
+POST_EXECUTION_CMDS = [
+    "sudo systemctl start cron",
```

Comment:
Random thought: shall we mention the `|| true` hack to allow things to fail?

Reply:
Good idea.